activities/SharedNotes.activity/js/activity.js

```diff
@@ -711,16 +711,13 @@ define(["sugar-web/activity/activity", "sugar-web/datastore", "notepalette", "zo
 					tutorial.start(tutorial.tourInit);
 				}, 500);
 			}
+				var defaultLanguage = (typeof chrome != 'undefined' && chrome.app && chrome.app.runtime) ? chrome.i18n.getUILanguage() : navigator.language;
+				var language = environment.user ? environment.user.language : defaultLanguage;
+        		l10n.init(language);
```

Comment:
Could you fix the indentation? Should be align and use TAB

Reply:
Sure I'll fix it give me 10-15 min

Reply:
![image](https://github.com/llaske/sugarizer/assets/93429474/706a7222-46ce-40ba-a363-9fcd8b8b2619)
it was looking good in my VS code but as per your suggestion added the tab